Monika After Story/game/script-topics.rpy

```diff
@@ -17567,6 +17567,53 @@ label monika_ddlcroleplay:
     m 1kub "Just make sure to remember that those versions of me are just stories, ahaha~"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_root_privileges",
+            conditional=(
+                "mas_hasRootPrivileges()"
+            ),
+            action=EV_ACT_PUSH,
+        )
+    )
+
+# Ever had root access/admin rights?
+default persistent._mas_pm_had_root_privileges = False
```

Comment:
More apt name might be `_mas_pm_gave_root_privileges_before`, historically or in dialogue use, the var becomes a more clear conditional.

Plus it's marked `pm` so it implies it's a player model var, i.e. a player's actions